### PR TITLE
Post-release version bump, and update release checklist

### PR DESCRIPTION
**Pull request by GeorgesStavracas**

### CODE REVIEW

**Review by ebassi — COMMENTED** [review]
LGTM

For reference: post-release version bumps always keep Git to a newer version than the release archives, so that people can use a version check when depending on a development snapshot.

**Comment by TingPing** [comment]
No complaints from me.

**Comment by smcv** [comment]
> I would set the version to 1.15.12 and then have a pre-stable-release bump to 1.16.0. That allow us to have a provision for another dev release. Also still show it as dev release.

I agree.

File: doc/release-checklist.md [hunk 1]
```diff
@@ -2,7 +2,6 @@ Flatpak release checklist
 =========================
 
 * Update NEWS
-* Update version number in `meson.build`
```
[the remark this answers is not in the document]
Comment (GeorgesStavracas) [reply doc/release-checklist.md:5]: Sure. Applied

**Comment by GeorgesStavracas** [comment]
Thanks everyone

**Pull request merged**